just_curious_game_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule JustCurious.GameTest do
+  use JustCurious.BotCase
+
+  setup do
+    no_chain_questions()
+    {:ok, %{game: standard_new_game()}}
+  end
+
+  test "New game setup properly", %{game: game} do
+    assert game.name == "Durazno"
+    assert game.channel == "durazno"
+    assert Enum.sort(participants(game)) == Enum.sort([@user1, @user2, @user3, @user4])
```

Comment:
I think an idiomatic way of testing if the two lists contain the same unsorted elements is to:

```
assert participants(game)) -- [@user1, @user2, @user3, @user4] == []
```